localstack/services/cloudformation/provider_utils.py

```diff
@@ -53,7 +53,7 @@ def remove_none_values(obj):
 
 # FIXME: this shouldn't be necessary in the future
 param_validation = re.compile(
-    r"Invalid type for parameter (?P<param>\w+), value: (?P<value>\w+), type: <class '(?P<wrong_class>\w+)'>, valid types: <class '(?P<valid_class>\w+)'>"
+    r"Invalid type for parameter (?P<param>[\w.]+), value: (?P<value>\w+), type: <class '(?P<wrong_class>\w+)'>, valid types: <class '(?P<valid_class>\w+)'>"
```

Comment:
I'm not entirely sure what this separate `provider_utils.py` is for, but I assume the logic should be copied over.

Reply:
We'll probably be able to remove this from this specific file in the future, but in the meantime thanks for keeping it in sync!

Some context: The idea of the `provider_utils.py` file is actually to be a simplified utils package with no dependencies to the rest of the localstack code base (or any other external dependencies) so that we can package this together with a resource provider when uploading to AWS for testing.